src/platform/forms-system/src/js/review/submit-states/ClientError.jsx

```diff
@@ -15,7 +15,7 @@ export default function ClientError(props) {
   const scrollToError = () => {
     scroller.scrollTo(
       'errorScrollElement',
-      window.VetsGov?.scroll || {
+      window.Forms?.scroll || {
```

Comment:
https://github.com/department-of-veterans-affairs/vets-website/pull/18712

> Standardizing where scroll options (needed for testing) are stored in either window.Forms or window.VetsGov?


It was requested as feedback from the first PR when I started this initiative.

Reply:
Gotcha. I just checked on latest `master`, and there are 39 files (notably, tests) that still use `VetsGov`. You may have some other open PRs to remove these, but I'd like to make sure that `window.VetsGov` isn't used anywhere before we remove it.

<details>
<summary>File list</summary>

```
src/platform/startup/react.js
src/platform/landing-pages/dev-template.ejs
src/applications/pensions/helpers.jsx
src/platform/site-wide/index.js
src/applications/burials/helpers.jsx
src/platform/testing/e2e-puppeteer/auth.js
src/platform/testing/e2e-puppeteer/helpers.js
src/platform/testing/unit/mocha-setup.js
src/applications/edu-benefits/feedback-tool/helpers.js
src/platform/testing/e2e/form-helpers.js
src/platform/testing/e2e/helpers.js
src/platform/utilities/ui/index.js
src/applications/pensions/components/AdditionalSourcesField.jsx
src/applications/pensions/tests/helpers.unit.spec.jsx
src/platform/site-wide/mega-menu/README.md
src/applications/burials/tests/helpers.unit.spec.jsx
src/applications/claims-status/containers/DocumentRequestPage.jsx
src/platform/forms/save-in-progress/RoutedSavableApp.jsx
src/platform/forms/save-in-progress/FormSaved.jsx
src/platform/forms/save-in-progress/RoutedSavableReviewPage.jsx
src/platform/forms/save-in-progress/SaveFormLink.jsx
src/applications/discharge-wizard/components/FormQuestions.jsx
src/applications/claims-status/actions/index.jsx
src/applications/edu-benefits/tests/feedback-tool/helpers.unit.spec.js
src/applications/edu-benefits/tests/feedback-tool/01.feedback-tool-keyboard.e2e.spec.js
src/applications/personalization/dashboard/tests/helpers.js
src/applications/personalization/profile/components/personal-information/ContactInformationField.jsx
src/applications/personalization/profile/components/personal-information/ContactInformationEditView.jsx
src/applications/personalization/dashboard/tests/unit/healthcare-messaging.unit.spec.jsx
src/platform/user/profile/vap-svc/containers/AddressValidationView.jsx
src/platform/user/profile/vap-svc/util/local-vapsvc.js
src/applications/personalization/profile/tests/components/personal-information/PersonalInformation.delete-address.unit.spec.jsx
src/applications/personalization/profile/tests/components/personal-information/PersonalInformation.delete-telephone.unit.spec.jsx
src/applications/personalization/profile/tests/components/personal-information/PersonalInformation.edit-email-address.unit.spec.jsx
src/applications/personalization/profile/tests/components/personal-information/PersonalInformation.edit-telephone.unit.spec.jsx
src/applications/personalization/profile/tests/components/personal-information/PersonalInformation.delete-email-address.unit.spec.jsx
src/platform/user/profile/vap-svc/components/base/VAPServiceTransactionPending.jsx
src/platform/forms-system/src/js/review/submit-states/ClientError.jsx
src/applications/personalization/profile/tests/components/personal-information/PersonalInformation.update-address.unit.spec.jsx
```

</details>

Here's an example:
https://github.com/department-of-veterans-affairs/vets-website/blob/d1827189e9ee5df69b6de66442612af967a01ac4/src/applications/personalization/profile/tests/components/personal-information/PersonalInformation.edit-telephone.unit.spec.jsx#L209

Reply:
Totally understand the concern, however I haven't removed the existence of `VetsGov` here. I've only eliminated the dependency on it for sourcing scrolling options, from a duplicate scrolling options function. I've eliminated the duplicate function, created one central one that sources off `window.Forms`, and redirected any scrolling-related references to the now eliminated function to the new one. This will allow us to now point to this scroll options function, in lieu of all the other conditional scrolling options we had discussed in other PRs.

Reply:
Just pushed an update that should clear it up a bit more as well. Once this is merged, I'd like to apply this same update to the other two PRs we've been discussing, to remove the necessity for a conditional in each of the individual components.